tests/backup/backup_rbac_test.go

```diff
@@ -482,12 +706,33 @@ var _ = Describe("{VerifyRBACForInfraAdmin}", func() {
 	})
 	JustAfterEach(func() {
 		defer EndPxBackupTorpedoTest(scheduledAppContexts)
-		nonAdminCtx, err := backup.GetNonAdminCtx(infraAdminUser, commonPassword)
+		backupDriver := Inst().Backup
+		bkpScheduleEnumerateReq := &api.BackupScheduleEnumerateRequest{
+			OrgId: orgID,
+		}
+		nonAdminCtx, err := backup.GetNonAdminCtx(customUser, commonPassword)
+		log.FailOnError(err, "failed to fetch user %s ctx", customUser)
+		currentSchedulesForCustomUser, err := backupDriver.EnumerateBackupSchedule(nonAdminCtx, bkpScheduleEnumerateReq)
+		log.FailOnError(err, "Getting a list of all schedules for Custom user")
+		for _, sch := range currentSchedulesForCustomUser.GetBackupSchedules() {
+			err = DeleteSchedule(sch.Name, SourceClusterName, orgID, nonAdminCtx)
```

Comment:
get cluster name from sch object?

Reply:
Making use of sch.Cluster instead of SourceClusterName